haiku-games/deathchase3d/deathchase3d-0.9.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A 3D shooter game"
+DESCRIPTION="DeathChase 3D is a game where a player controls a \
+motorcycle-riding mercenary as he pursues two other motorcycles \
+through a forest. Your goal is to try to destroy these motorcycles \
+for points, but try to avoid hitting the trees as you go. Control \
+the motorcycle with Z,X,K, and M, press space to fire, and ESC to exit."
+HOMEPAGE="https://osgameclones.com/"
+COPYRIGHT="2002 Paul Robson"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://web.archive.org/web/20070711231311/http://www.robsons.org.uk/archive/www.autismuk.freeserve.co.uk/deathchase3d-0.9.tar.gz"
+CHECKSUM_SHA256="42b4c5551be4662fcba6a6c7476ff6235307cf39e13781d6f1f13abe26a7db2a"
+SOURCE_DIR="deathchase3d-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	deathchase3d = $portVersion
+	app:deathchase3d = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	"
+BUILD_PREREQUIRES="
+	cmd:ld
+	cmd:libtoolize
+	cmd:g++
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -E 's|(\$\(top_srcdir\)/)?admin/libtool.m4.in||g' -i Makefile.am
```

Comment:
You can add the code that deals with regenerating `acinclude.m4` and then the entire patchset could be dropped.